brvs-core/src/main/java/iroha/validation/rest/RestService.java

```diff
@@ -77,11 +80,31 @@ public Response register(@PathParam("accountId") String accountId) {
   @Consumes(MediaType.APPLICATION_JSON)
   @Produces(MediaType.APPLICATION_JSON)
   public Response sendTransaction(String transaction) {
+    return sendTransaction(transaction, false);
+  }
+
+  @POST
+  @Path("/transaction/sign")
```

Comment:
I'd rather call it `signAndSend`.

Reply:
cmon camel case urls are awful

Reply:
I agree.

Reply:
Could you please rename your URI to make its purpose more clear?